### DISCUSSION TITLE
Missing classes after building css

### DISCUSSION THREAD

**Discussion by jaakkoijas** [tailwindlabs/tailwindcss#9947] [Help] [answered]
What is wrong with my tailwind setup when certain classes aren't available? Example `bg-red-500`. When I search it from builded css file, there is only bg-red-100, -600 and -700. Also every top- class are missing, but left, right and bottom are there.

I have run `npm run dev` command and it is working because I can see the css file changing.

Tailwind version is 3.2.4

Here is my tailwind.config.js:

```
module.exports = {
    content: [
        './storage/framework/views/*.php',
        './resources/views/**/*.blade.php'
    ],

    plugins: [
        require('@tailwindcss/forms'),
        require('@tailwindcss/typography'),
    ],
};
```

**Comment by wongjn** [comment, marked as answer]
The Tailwind configuration looks fine. Some other things to check:
- The files for any code where you use `bg-red-500` and where you use any `top-` classes are sufficiently covered by the `content` file globs.
- String interpolation or concatenation is not being used to build the class names.

**Reply by jaakkoijas** [reply]
Thank you. I had to add my subfolder to config->content section.  I thought that `./resources/views/**/*.blade.php` would have covered all folders and subfolders.
